### PR TITLE
Feature/cv_clara

### PR DESCRIPTION
**Pull request by CRR257**

### CODE REVIEW

File: staff/clara-rufi/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Clara Rufí</title>
+</head>
+    <header>
+    <h1>Clara Rufí</h1>
+    <img>
+    <p></p>
```
Comment (robert-z) [inline staff/clara-rufi/cv/index.html]: If you want a newline, I think better `<br/>` tag

File: staff/clara-rufi/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <div id= "personal information:"></div>
+    <div id= "informationP">Age: 33</div>
+    <div id= "informationP">ID: 47732058C</div>
+    <div id= "informationP">mobile: 617555958</div>
+    <div id= "informationP">email: clara_rufi@hotmail.com</div>
+    <a href="www.clararufi.com">www.clararufi.com</a>
+    </header>
+
+    <h3>Education:</h3>
+    <ol>2017. Training certificate Training certificate BacGene Salmonella spp -PCR kit for the detection of Salmonella spp. </ol>
```
Comment (robert-z) [inline staff/clara-rufi/cv/index.html]: Example for `<ol>` element. Use the `<li>` tag to define list items.
```html
<ol>
  <li>Coffee</li>
  <li>Tea</li>
  <li>Milk</li>
</ol>
```

File: staff/clara-rufi/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    <ol>2017. GMP - Quality system of the pharmaceutical sector</ol>
+    <ol>2007-2010. Science and Food Technology. UB</ol>
+    <ol>2003-2007. Nutrition. UVic</ol>
+
+    <p></p>
+
+    <h3>Work experience:</h3>
+    <h4><ol> 2010 - 2012 Bactereco. Analista d’aliments i d’aigües. Sant Feliu de Llobregat</h4></ol>
+        <p>
+            <ul>- Microbiological analysis of food and water. Expression of results. Traceability of all the means and utensils used.</ul>
```
Comment (robert-z) [inline staff/clara-rufi/cv/index.html]: Example for `<ul>` element. Use the `<li>` tag to define list items.
```html
<ul>
  <li>Coffee</li>
  <li>Tea</li>
  <li>Milk</li>
</ul>
```

File: staff/clara-rufi/cv/index.html (hunk 1 above)
Comment (manuelbarzi) [inline staff/clara-rufi/cv/index.html]: empty `<p>` has no sense here. remove unused or unnecessary tags.

File: staff/clara-rufi/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    <a href="www.clararufi.com">www.clararufi.com</a>
+    </header>
+
+    <h3>Education:</h3>
+    <ol>2017. Training certificate Training certificate BacGene Salmonella spp -PCR kit for the detection of Salmonella spp. </ol>
+    <ol>2017. GMP - Quality system of the pharmaceutical sector</ol>
+    <ol>2007-2010. Science and Food Technology. UB</ol>
+    <ol>2003-2007. Nutrition. UVic</ol>
+
+    <p></p>
```
Comment (manuelbarzi) [inline staff/clara-rufi/cv/index.html]: same here, empty `<p>` has no sense, no meaning. remove it.

File: staff/clara-rufi/cv/index.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+    <h3>Work experience:</h3>
+    <h4><ol> 2010 - 2012 Bactereco. Analista d’aliments i d’aigües. Sant Feliu de Llobregat</h4></ol>
```
Comment (robert-z) [inline staff/clara-rufi/cv/index.html]: `<ol>` tag is not necessary

Comment (manuelbarzi) [inline staff/clara-rufi/cv/index.html]: RTFM https://www.w3schools.com/tags/tag_ol.asp

File: staff/clara-rufi/cv/index.html (hunk 1 above)
Comment (CRR257) [reply staff/clara-rufi/cv/index.html]: thanks, I have corrected it

File: staff/clara-rufi/cv/index.html (hunk 2 above)
Comment (CRR257) [reply staff/clara-rufi/cv/index.html]: Thanks

File: staff/clara-rufi/cv/index.html (hunk 5 above)
Comment (CRR257) [reply staff/clara-rufi/cv/index.html]: ok, thanks

File: staff/clara-rufi/cv/index.html (hunk 1 above)
Comment (CRR257) [reply staff/clara-rufi/cv/index.html]: ok, I have corrected it

**Comment by manuelbarzi** [comment]
@CRR257 when you apply a correction, remember to resolve the conversation related to it. please do it for the conversations above, the ones you have already taken action.

**Pull request merged**